data/conf/samples/ers_mysql_update/cgrates.json

```diff
@@ -0,0 +1,84 @@
+{
+
+	"general": {
+		"log_level": 7,
+	},
+	
+	"apiers": {
+		"enabled": true
+	},
+	"filters": {			
+		"apiers_conns": ["*localhost"]
+	},
+	"stor_db": {
+		"opts": {
+			"sqlConnMaxLifetime": "5s", // needed while running all integration tests
+		},
+	},
+	"ees": {
+		"enabled": true,
+		"exporters": [{
+			"id": "SQLExporter",
+			"type": "*sql",
+			"export_path": "mysql://cgrates:CGRateS.org@127.0.0.1:3306",
+			"attempts": 1,
+			"opts": {
+				"sqlDBName": "cgrates2",
+				"sqlTableName":"cdrs", 
+				"sqlUpdateIndexedFields": ["id", "cgrid"],
+			},
+			"filters":["*string:~*req.ReaderID:mysqlReaderID"],
+			"flags": ["*log"],
+			"fields": [
+				{"tag": "SetupTime", "path": "*exp.setup_time", "type": "*constant", "value": "2018-11-27 14:21:26"},
+				{"tag": "Account", "path": "*exp.account", "type": "*variable", "value": "~*req.ExtraInfo"},
+				{"tag": "ID", "path": "*exp.id", "type": "*variable", "value": "~*req.Id"},
+				{"tag": "CGRID", "path": "*exp.cgrid", "type": "*variable", "value": "~*req.CGRID"},
+			]
+		}]
+	},
+	"ers": {									
+		"enabled": true,						
+		"sessions_conns":["*localhost"],
+		"ees_conns": ["*localhost"],
+		"readers": [{
+				"id": "mysqlReaderID",										
+				"type": "*sql",				
+				"run_delay": "1m",									
+				"source_path": "*mysql://cgrates:CGRateS.org@127.0.0.1:3306",					
+				"opts": {
+					"sqlDBName":"cgrates2",
+					"sqlTableName":"cdrs", 
+				},
+				"start_delay": "500ms", // wait for db to be populated before starting reader 
+				"tenant": "cgrates.org",							
+				"filters": [
+				"*gt:~*req.answer_time:-168h", // dont process cdrs with answer_time older than 7 days ago 
```

Comment:
Small note here, the best would be to indent the filters with a tab or something since they are on the same level.